QA_Requests/Vaccine/OutputFiles/Forms/Results1.csv

```diff
@@ -0,0 +1,140 @@
+ca1c973427564967a65b689c80caedb3,5660ac09-5589-403f-a96a-bce5a8d2199e,429
+ca1c973427564967a65b689c80caedb3,fbd82072-2df4-4f22-afd6-ee89bae7b11a,429
+ca1c973427564967a65b689c80caedb3,e15aad9d-5b2a-43ee-9f56-c4e2e81ea7a2,429
+ca1c973427564967a65b689c80caedb3,2ccec62e-2fb8-47ef-82e2-75f9366a5d62,429
+ca1c973427564967a65b689c80caedb3,e42a878e-b30b-47c0-87b0-57a20e48c288,429
+ca1c973427564967a65b689c80caedb3,a096c0e1-83bd-48a2-b9b7-72843b64349d,429
+ca1c973427564967a65b689c80caedb3,70d93b05-91e0-4380-b1ea-d04398859b6d,429
+ca1c973427564967a65b689c80caedb3,03aa435e-4382-441d-85a5-0417ec0a4b28,429
+ca1c973427564967a65b689c80caedb3,547be0d9-fdc1-4516-97aa-03094fa21fee,429
+ca1c973427564967a65b689c80caedb3,502fc2b3-5518-41ef-bf7f-0b8ba3645ce7,429
+ca1c973427564967a65b689c80caedb3,eba7a9b4-a580-497f-b96f-67f224c46f4b,429
+ca1c973427564967a65b689c80caedb3,503b1028-06ab-43e4-8e33-e09f38afe749,429
+ca1c973427564967a65b689c80caedb3,df5916dc-97ae-4951-b934-a652aea02c61,429
+ca1c973427564967a65b689c80caedb3,952ff814-2708-472f-ac9c-ce397832304a,429
+ca1c973427564967a65b689c80caedb3,11a52d99-7fda-426a-97e3-4b8bec5979d5,429
+ca1c973427564967a65b689c80caedb3,57f0b97a-fc56-41a4-888b-9da252aa4695,429
+ca1c973427564967a65b689c80caedb3,b9cb833b-3677-4ec0-b6b6-82c10808058b,429
+ca1c973427564967a65b689c80caedb3,6f47b382-ed9e-49af-9dd0-a9e1579e826d,429
+ca1c973427564967a65b689c80caedb3,693daa36-759f-484b-9d97-b474e82e74b2,429
+ca1c973427564967a65b689c80caedb3,05592bb8-41b1-4ad4-ac0c-a072c60f331a,429
+ca1c973427564967a65b689c80caedb3,38d1019d-bac7-4641-8e6b-d6ce9886e979,429
+ca1c973427564967a65b689c80caedb3,b6341b26-23de-4c8e-bd27-55412eaca0fa,429
+ca1c973427564967a65b689c80caedb3,277c2197-da73-4a64-810e-6ec79faf335f,429
+ca1c973427564967a65b689c80caedb3,593a9e94-e9d6-4a4a-b938-5c1914cd25c2,429
+ca1c973427564967a65b689c80caedb3,90904d57-caed-49ed-9501-48005e3ad5ee,429
+ca1c973427564967a65b689c80caedb3,dbd34eb3-1fc9-4ebb-9756-c7fffaf6a759,429
+ca1c973427564967a65b689c80caedb3,df075157-f49c-4a7b-aea6-2840f37c7afe,429
+ca1c973427564967a65b689c80caedb3,9f118684-eea3-440b-abcb-eb8b1b46a429,429
+ca1c973427564967a65b689c80caedb3,c848ede2-bac8-449a-81d3-05dc54119285,429
+ca1c973427564967a65b689c80caedb3,4c11e397-9d13-4617-a366-c1c578c08f34,429
+ca1c973427564967a65b689c80caedb3,a23b1be3-6a3e-4195-b045-b008bb82283f,429
+ca1c973427564967a65b689c80caedb3,af34302e-6368-41e5-8f7a-fc38c68da681,429
+ca1c973427564967a65b689c80caedb3,bbfc61bb-4393-4871-ab76-f689d5dbdf0b,429
+ca1c973427564967a65b689c80caedb3,15b5cced-96e8-46c5-ac5a-ebc46b3af193,429
+ca1c973427564967a65b689c80caedb3,fac3f242-d85a-47bf-8176-7311f12da0ec,429
+ca1c973427564967a65b689c80caedb3,16cb30cb-c6e0-43a6-b8e0-71b542cf28cc,429
+ca1c973427564967a65b689c80caedb3,0daf51e4-64e5-4c57-9bcf-81b3fbf6a197,429
+ca1c973427564967a65b689c80caedb3,31e8d27f-783f-4fbd-a9f5-0fd0b44a40b4,429
+ca1c973427564967a65b689c80caedb3,f87574fd-c26d-4d93-aa1e-dd820af17fec,429
+ca1c973427564967a65b689c80caedb3,8a7eb691-66f7-469c-ae87-3bfe1924a7c7,429
```

Comment:
Same goes for these outputs as well.

Reply:
So it'd be fine to delete the complete directory `OutputFiles`?

Reply:
I guess so. @kiran-tripathy

Reply:
Deleted!